cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -0,0 +1,529 @@
+                .withType("RollingUpdate")
+                .withRollingUpdate(new RollingUpdateDeploymentBuilder()
+                        .withMaxSurge(new IntOrString(1))
+                        .withMaxUnavailable(new IntOrString(0))
+                        .build())
+                .build();
+
+        return createDeployment(
+                updateStrategy,
+                Collections.emptyMap(),
+                annotations,
+                getMergedAffinity(),
+                getInitContainers(imagePullPolicy),
+                getContainers(imagePullPolicy),
+                getVolumes(isOpenShift),
+                imagePullSecrets);
+    }
+
+    @Override
+    protected List<Container> getContainers(ImagePullPolicy imagePullPolicy) {
+
+        List<Container> containers = new ArrayList<>();
+
+        Container container = new ContainerBuilder()
+                .withName(name)
+                .withImage(getImage())
+                .withCommand("/opt/strimzi/bin/docker/kafka_bridge_run.sh")
+                .withEnv(getEnvVars())
+                .withPorts(getContainerPortList())
+                .withLivenessProbe(ModelUtils.createHttpProbe(livenessPath, HEALTH_CHECK_PORT_NAME, livenessProbeOptions))
+                .withReadinessProbe(ModelUtils.createHttpProbe(readinessPath, HEALTH_CHECK_PORT_NAME, readinessProbeOptions))
+                .withVolumeMounts(getVolumeMounts())
+                .withResources(getResources())
+                .withImagePullPolicy(determineImagePullPolicy(imagePullPolicy, getImage()))
+                .build();
+
+        containers.add(container);
+
+        return containers;
+    }
+
+    @Override
+    protected List<EnvVar> getEnvVars() {
+        List<EnvVar> varList = new ArrayList<>();
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_METRICS_ENABLED, String.valueOf(isMetricsEnabled)));
+
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_BOOTSTRAP_SERVERS, bootstrapServers));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_CONSUMER_CONFIG, kafkaBridgeConsumer == null ? "" : new KafkaBridgeConsumerConfiguration(kafkaBridgeConsumer.getConfig().entrySet()).getConfiguration()));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_PRODUCER_CONFIG, kafkaBridgeProducer == null ? "" : new KafkaBridgeProducerConfiguration(kafkaBridgeProducer.getConfig().entrySet()).getConfiguration()));
+
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_HTTP_ENABLED, String.valueOf(httpEnabled)));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_HTTP_HOST, KafkaBridgeHttpConfig.HTTP_DEFAULT_HOST));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_HTTP_PORT, String.valueOf(http != null ? http.getPort() : KafkaBridgeHttpConfig.HTTP_DEFAULT_PORT)));
+
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_BRIDGE_AMQP_ENABLED, String.valueOf(amqpEnabled)));
+        //TODO amqp configuration
+
+        heapOptions(varList, 1.0, 0L);
```

Comment:
As mentioned above, the `heapOptions` and `jvmPerformanceOptions` might not work outside of Kafka pods. But it might make sense to add this instead of them:

```
varList.add(buildEnvVar(ENV_VAR_STRIMZI_GC_LOG_ENABLED, String.valueOf(gcLoggingEnabled)));
```